pkg/minishift/util/setproxytoshell.go

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
@budhrg looks like in file name can have '_' but http://stackoverflow.com/a/25162021 should also need to consider.